<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DailyRemoval</class>
 <widget class="QMainWindow" name="DailyRemoval">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DailyRemoval</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>601</width>
      <height>801</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="startImage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>601</width>
       <height>801</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix/start.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="titleLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>401</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix/title.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="instructionButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>490</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Forte</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(207, 184, 52);</string>
     </property>
     <property name="text">
      <string>Instruction</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quitButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>630</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Forte</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(207, 184, 52);</string>
     </property>
     <property name="text">
      <string>quit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>350</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Forte</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(207, 184, 52);
</string>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="musicPlayButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix/soundOn.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="musicPlayButton_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix/soundOff.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>startImage</zorder>
    <zorder>titleLabel</zorder>
    <zorder>playButton</zorder>
    <zorder>quitButton</zorder>
    <zorder>instructionButton</zorder>
    <zorder>musicPlayButton</zorder>
    <zorder>musicPlayButton_2</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
